src/state_title_screen.c

```diff
@@ -73,7 +73,7 @@ typedef struct TitleDataStruct {
 extern s16 D_800A0970;
 extern TitleDataStruct* TitleScreen_ImgList;
 extern s32* TitleScreen_ImgList_Logo;
-extern s32* TitleScreen_ImgList_Copyright;
+extern u8 (*TitleScreen_ImgList_Copyright)[144]; // 144 being the width of the image
```

Comment:
144 should be a constant. The JP version uses a width of 128
https://github.com/pmret/papermario/blob/main/tools/splat_ext/pm_map_data.py#LL141C1-L147C62